src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -18,15 +21,27 @@
 public class SampleDataUtil {
 
     public static Recipe[] getSampleRecipes() {
-        return new Recipe[]{
```

Comment:
Made this less cancerous. My eyes bled while reading the original code. Hopefully the new one is neater.